src/marvin/bot/base.py

```diff
@@ -35,6 +35,7 @@ class BotResponse(BaseMessage):
 if TYPE_CHECKING:
     from marvin.models.bots import BotConfig
 DEFAULT_NAME = "Marvin"
+DEFAULT_DESCRIPTION = ""
```

Comment:
```suggestion
DEFAULT_DESCRIPTION = "A Marvin bot"
```

Reply:
Let's put some basic content here to show something as long as we have the opportunity?